Chapters/04-implicitni-funkce.tex

```diff
@@ -50,6 +50,83 @@ \subsection{Věty o implicitní funkci}
 	$f: (x_0 - \delta , x_0 + \delta ) \to \mathbb{R}$ má spojité derivace do řádu $k$.
 \end{theorem}
 
+\begin{proof}
+	Buď třeba
+	$$
+	\frac{\partial F(x_0, y_0)}{\partial y} > 0
+	$$
+	Vzhledem ke spojitosti existují $\delta_1, \Delta > 0$ taková, že v
+	obdélníku $\langle x_0 - \delta_1, x_0 + \delta_1 \rangle \times
+	\langle y_0 - \Delta, y_0 + \Delta \rangle$ je $\frac{\partial
+	F}{\partial y} (x, y)$ stále kladná (víme, že $\exists \epsilon:
```

Comment:
It would be better to use `\varepsilon` instead of `\epsilon` in most cases. Mostly for consistency, but also because (imo) it is a tad more readable.